solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/list.test.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('list environment type configs', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+  const envTypeId = setup.getSettings().get('envTypeId');
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  test('list environments type configs when filter and sorting by name', async () => {
+    const { data: response } = await adminSession.resources.environmentTypeConfigs.get({}, envTypeId);
```

Comment:
Is this correct? The test name mentions `filter and sorting by name`, but I didn't see `name` mentioned in the request.

Reply:
this was a copy paste issue, I will fix the description, ETC does not have any filter

Reply:
fixed